docs/plugins/testing_workshop_docs/Readers-and-fixtures.md

```diff
@@ -0,0 +1,147 @@
+This article will be more valuable if you are familiar with the Python programming language and the Napari software. It is the third in a series of articles on testing taken from the [January 2022 testing workshop video](https://drive.google.com/file/d/1DaMrRz-rLRQ6-_y0J8O3GRpVPCn0rgYs/view). The information in this article starts at minute 15:42. This article should stand on its own and is a summary of the information in the video. The other articles are:  
```

Comment:
You need a title for every file:

```suggestion
# Readers and fixtures

This article will be more valuable if you are familiar with the Python programming language and the Napari software. It is the third in a series of articles on testing taken from the [January 2022 testing workshop video](https://drive.google.com/file/d/1DaMrRz-rLRQ6-_y0J8O3GRpVPCn0rgYs/view). The information in this article starts at minute 15:42. This article should stand on its own and is a summary of the information in the video. The other articles are:  
```

Reply:
Melissa, this was taken out of the individual articles and put in the Frontpage article (index.md). This makes more sense than repeating it everywhere.  I hope you like it.